nixos/doc/manual/release-notes/rl-2305.section.md

```diff
@@ -132,6 +132,8 @@ In addition to numerous new and upgraded packages, this release has the followin
   [upstream's release notes](https://github.com/iputils/iputils/releases/tag/20221126)
   for more details and available replacements.
 
+- The new option `services.caddy.enableReload` is enabled by default, and allows reloading the Caddy service instead of restarting it if a config file is the only thing that changes. This option must be disabled if you have turned off the [Caddy admin API](https://caddyserver.com/docs/caddyfile/options#admin). If you keep this option enabled, you should consider setting [`grace_period`](https://caddyserver.com/docs/caddyfile/options#grace-period) to a non-infinite value to prevent Caddy from delaying the reload indefinitely.
```

Comment:
```suggestion
- The Caddy module gained a new options named `services.caddy.enableReload` which is enabled by default and allows reloading the service instead of restarting if only a config file changed. This option must be disabled if you have turned off the [Caddy admin API](https://caddyserver.com/docs/caddyfile/options#admin). If you keep this option enabled, you should consider setting [`grace_period`](https://caddyserver.com/docs/caddyfile/options#grace-period) to a non-infinite value to prevent Caddy from delaying the reload indefinitely.
```
If this is enabled by default, we should set a grace_period, too

Reply:
Is this usage of `if only` right? OALD says "if only" is used to say that you wish sth was true or that sth had happened.  Any comments for native English speakers?

Reply:
Yes, it sounds right to me - "only" here is a qualifier for the config file change. I think the sentence is doing too much work though, and would be better split up into two sentences:

> The Caddy module gained a new option named `services.caddy.enableReload` which is enabled by default. It allows reloading the service instead of restarting it, if only a config file has changed.

This corrects "options" to "option" and adds some words to the second sentence to make it read more easily.